modules/virt-creating-vm-web-page-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * virt/creating_vms_advanced/creating_vms_advanced_web/virt-creating-vms-from-web-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-vm-import-cli_{context}"]
+= Creating a VM from an image on a web page by using the command line
+
+You can create a virtual machine (VM) from an image on a web page by using the command line.
+
+When the virtual machine (VM) is created, the data volume with the image is imported into persistent storage.
+
+.Prerequisites
+
+* You must have access credentials for the web page that contains the image.
+* You must have the `virtctl` command line tool installed.
+
+.Procedure
+
+. Create a `VirtualMachine` manifest for your VM and save it as a YAML file. For example, to create a minimal {op-system-base-full} VM from an image on a web page, run the following command:
++
+[source,terminal]
+----
+$ virtctl create vm --name vm-rhel-9 --instancetype u1.small --preference rhel.9 --volume-import type:http,url:https://example.com/rhel9.qcow2,size:10Gi
+----
+
+. Review the `VirtualMachine` manifest for your VM:
++
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-rhel-9 # <1>
+spec:
+  dataVolumeTemplates:
+  - metadata:
+      name: imported-volume-6dcpf # <5>
```

Comment:
This threw me off that it went from callout 1 to 5. I think normally we'd want to see them go in numerical order. So I'd recommend updating the callout numbers here and below to go in order.

Reply:
That is a relic from when the module overused conditions, I will re-order the callouts.